### PR TITLE
Int 1103 add pre annotations support in input api client

### PR DESCRIPTION
**Pull request by alydh-annotell**

### CODE REVIEW

**Review by petern-annotell — APPROVED** [review]

File: annotell-input-api/tests/lidars_and_cameras_seq_with_pre_annotations_test.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+from tests.utils import TestProjects
+
+
+class TestLidarsAndCamerasSeqWithPreAnnotations:
+
+    @staticmethod
+    def filter_lidar_and_cameras_seq_project(projects: List[IAM.Project]):
+        return [p for p in projects if p.project == TestProjects.LidarsAndCamerasSequenceProject]
+
+    def test_validate_lidars_cameras_seq_with_imu_and_shutter_times_example(self, client: IAC.InputApiClient):
```
[the remark this answers is not in the document]
Comment (alydh-annotell) [reply annotell-input-api/tests/lidars_and_cameras_seq_with_pre_annotations_test.py:19]: No i'll update them

**Comment by alydh-annotell** [comment]
Ran this towards the test setup in staging and it creates inputs with preannotations

File: annotell-input-api/annotell/input_api/model/input/preannotations/validated_pre_annotations.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+from typing import Optional
```
Comment (olinoskar) [inline annotell-input-api/annotell/input_api/model/input/preannotations/validated_pre_annotations.py:1]: remove file

File: annotell-input-api/annotell/input_api/model/input/preannotations/__init__.py [hunk 3]
```diff
@@ -0,0 +1 @@
+from .validated_pre_annotations import *
```
Comment (olinoskar) [inline annotell-input-api/annotell/input_api/model/input/preannotations/__init__.py:1]: remove file

File: annotell-input-api/examples/lidars_and_cameras_seq_with_pre_annotations.py [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+        calibration_id=created_calibration.id,
+        metadata=metadata
+    )
+
+    # Create Scene but not input since we don't provide project or batch
+    scene_response = client.lidars_and_cameras_sequence.create(lidars_and_cameras_seq, annotation_types=annotation_types, dryrun=dryrun)
+
+    # Simulate waiting for scene to be created since this is a async process
+    # The rest needs to be run once the scene creation has finished
+    time.sleep(10)
```
Comment (olinoskar) [inline annotell-input-api/examples/lidars_and_cameras_seq_with_pre_annotations.py:86]: add a print (or log) here to make it clear that we are sleeping and that nothing is wrong

**Review by olinoskar — CHANGES_REQUESTED** [review]
A few comments. We also need to add `annotell-openlabel` in setup.py now

**Review by olinoskar — APPROVED** [review]

**Pull request merged**